train.py

```diff
@@ -138,7 +138,7 @@ def __init__(self):
             variables_to_restore = [var for var in self.net_var if var.name.split(':')[0] in ckpt_net_var]
             self.loader = tf.train.Saver(variables_to_restore)
             # -nadav_wp_pruning
-            self.saver = tf.train.Saver(tf.global_variables(), max_to_keep=10)
+            self.saver = tf.train.Saver(tf.global_variables(), max_to_keep=20)
```

Comment:
no reason of modifying this line, consider providing max_to_keep as configuration parameter